sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -246,14 +240,6 @@ public void setUserGroup(UserGroupProperties userGroup) {
         this.userGroup = userGroup;
     }
 
-    public String getEnvironment() {
-        return environment;
-    }
-
-    public void setEnvironment(String environment) {
-        this.environment = environment;
-    }
-
```

Comment:
Can we delete `UserGroupProperties.key` and `UserGroupProperties.value` `UserGroupProperties.objectIDKey`?